arches_lingo/src/arches_lingo/App.vue

```diff
@@ -38,10 +39,21 @@ router.beforeEach(async (to) => {
 </script>
 
 <template>
-    <main style="font-family: sans-serif; height: 100vh">
-        <PageHeader v-if="route.meta.shouldShowHeader" />
-        <div style="margin: 1rem">
-            <RouterView />
+    <main
+        style="
+            font-family: sans-serif;
+            height: 100vh;
+            width: 100vw;
+            display: flex;
+            flex-direction: column;
+        "
```

Comment:
Without this in root.htm:
```
{% block body_attributes %}style="margin: 0"{% endblock %}
```

I get 8px of margin, causing scrolls (because I'm a pest and I configure MacOS to have Show Scroll bars > Always):



<img width="875" alt="margin" src="https://github.com/user-attachments/assets/bbf2d38e-c149-4c03-8226-fb75f41a6cef">

Reply:
OH! this is fixed by https://github.com/archesproject/arches/pull/11343. I've added it manually to my local and have been developing against it 👍

Reply:
Hmm, I tried that and it didn't help but I'll check again tomorrow maybe I botched the build.

Reply:
no need to wait for me, let's merge 👌